guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,29 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using Linux Unified Key Setup (LUKS) during host provisioning.
+Optionally, you can also bind the disk to a Tang server through Clevis for decryption during boot.
+You must associate the partition table with your operating system entry.
+Then, you assign the partition table to your host group or select it manually during provisioning.
+
+Encrypted disks on hosts protect data at rest.
```

Comment:
This sentence seems out of place here, isolated. I thought about a way to reorganize the introduction to move it to a place where it would have context. This is what I came up with:

> 
> {Project} contains partition tables that encrypt the disk of your host by using Linux Unified Key Setup (LUKS) during host provisioning.
> Encrypted disks on hosts protect data at rest.
> Optionally, you can also bind the disk to a Tang server through Clevis for decryption during boot.
> 
> Associate the partition table with your operating system entry.
> Then, you assign the partition table to your host group or select it manually during provisioning.

Feel free to apply, adjust, or discard :)

Reply:
Excellent, applied!